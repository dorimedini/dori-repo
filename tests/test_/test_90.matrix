{
    int a = 0;
    foreach (b in [1,2;3,4;5,6]){
        print b;
        a = a+b;
    }
    print a;

    int i;
    int n1 = 0,n2 = 1, temp;
    while (i < 10){
        temp = n1+n2;
        n1 = n2;
        n2 = temp;
        print n2;
        i = i+1;
    }

    i = 0;
    while (i < 5){
        foreach (i in [1,10;2,20;3,30]){
            a = i*i;
            print a;
        }
        i = i+1;
    }
}
