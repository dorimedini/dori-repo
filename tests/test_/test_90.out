1
2
3
4
5
6
21
1
2
3
5
8
13
21
34
55
89
1
100
4
400
9
900
1
100
4
400
9
900
1
100
4
400
9
900
1
100
4
400
9
900
1
100
4
400
9
900
